deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -534,95 +387,107 @@ data:
         relabel container_name:container,pod_name:pod
       </filter>
       
+  
+    
       <match prometheus.metrics.apiserver**>
         @type sumologic
         @id sumologic.endpoint.metrics.apiserver
-        sumo_client "k8s_1.0.0"
```

Comment:
Why this is deleted?
Would you mind rebasing on current origin/master and deleting autogenerated commits (so that they can be regenerated once more)?

Reply:
I think something with rebase goes wrong

Reply:
solved in #672

Reply:
`<<<<<<< HEAD
=======` above is more interesting